org.lflang/src/org/lflang/generator/LFGenerator.java

```diff
@@ -128,12 +128,12 @@ private GeneratorBase createKotlinBaseGenerator(Target target, FileConfig fileCo
             generatorErrorsOccurred = true;
             errorReporter.reportError(
                 "The code generator for the " + target + " target could not be found. "
-                    + "This is likely because you are running the RCA from"
-                    + "Eclipse. The " + target + " code generator is written in Kotlin"
-                    + "and, unfortunately, the Eclipse Kotlin plugin is "
-                    + "broken, preventing us from loading the generator"
-                    + "properly. Please consider building the RCA via Maven.\n"
-                    + "For instructions building RCA via Maven, see: "
+                    + "This is likely because you built Epoch using "
+                    + "Eclipse. The " + target + " code generator is written in Kotlin "
+                    + "and, unfortunately, the plugin that Eclipse uses "
+                    + "for compiling Kotlin code, is broken. "
```

Comment:
Comma is not needed here.

Reply:
Where? Multiple comma's there :-)

Reply:
Sorry... I thought the comment would attach to a specific line.  The comma before "is broken"  is wrong.